con-master/src/main/java/ba/codecentric/master/controller/HotelController.java

```diff
@@ -19,7 +23,15 @@ public HotelController(HotelService hotelService) {
     }
 
     @RequestMapping(value = "/hotel/save", method = RequestMethod.POST)
-    public void saveHotel(@RequestBody Hotel hotel) {
-        hotelService.saveHotel(hotel);
+    public ResponseEntity<Hotel> saveHotel(@RequestBody Hotel hotel) {
```

Comment:
Use RESTful approach. There's no need for "/hotel/save". You'd probably want to have only "hotel" or "hotels" as a URI. Same question as in comment above. Is there a need for ResponseEntity?